### PR TITLE
Application of LangCaption in German

### PR DESCRIPTION
**Pull request by NeonKnightOA**
Requesting review before committing.
Tagging @SlyMk3 because I couldn't ask him for a review.

### CODE REVIEW

**Review by JanUrbansky — COMMENTED** [review]
Don't "verhackstück" those words. I believe "Commandlets und Exporter" is enough. "Exporter" or "Exporters" are also spelled in german "Exporter" not "Exporteur".
I would not translate "Commandlet"

**Review by JanUrbansky — COMMENTED** [review]
My suggestion:

Konfig -> Konfiguration
Klient -> Client

"fur" -> "für"

**Review by JanUrbansky — COMMENTED** [review]
My suggestion:

"Deko Schatten" -> "Dekorationsschatten"
"Projektil Schatten" -> "Projektilschatten"

"Tastenfunktionen" -> "Tastenbelegung"
"Schaltflächenbefehle" -> "Tastenkürzel"

"Kanal herunterladen" -> "Kanal Download"

**Review by JanUrbansky — COMMENTED** [review]
"Mit Hauptserver verbinden" -> "Hauptserver Verbindung" or "Master Server Verbindung"

**Review by JanUrbansky — COMMENTED** [review]
fur -> für in "Karten fur Düster-Match"

File: Unreal/SystemLocalized/det/UnrealI.det [hunk 1]
```diff
@@ -349,11 +349,11 @@ Message7="Drücke Feuer zum Neustarten"
 
 [KingOfTheHill]
 ; EN: ClassCaption="King of the Hill"
-ClassCaption="King of the Hill"
+ClassCaption="König des Hügels"
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/UnrealI.det:352]: I intentionally did not translate this from "King of the Hill", as thats a fairly well known English term and a common game mode in video games. I've yet to see anyone say or write "König des Hügels" anywhere.

File: Unreal/SystemLocalized/det/UnrealIntegrity.det [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿[Public]
-Preferences=(Caption="Unreal Integrity",Parent="Networking")
-Preferences=(Caption="Integrity Server",Parent="Unreal Integrity",Class=UnrealIntegrity.IntegrityServer,Immediate=True)
-Preferences=(Caption="Whitelist",Parent="Unreal Integrity",Class=UnrealIntegrity.ExternalWhiteList,Immediate=True)
+Preferences=(Caption="Unreal Integrity",Parent="Networking",LangCaption="Integrität von Unreal")
+Preferences=(Caption="Integrity Server",Parent="Unreal Integrity",LangCaption="Integritätsserver",Class=UnrealIntegrity.IntegrityServer,Immediate=True)
+Preferences=(Caption="Whitelist",Parent="Unreal Integrity",LangCaption="Weiße Liste",Class=UnrealIntegrity.ExternalWhiteList,Immediate=True)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/UnrealIntegrity.det:4]: "Whitelist" is perfectly fine as a German term, nobody says Weiße Liste.

File: Unreal/SystemLocalized/det/XOpenGLDrv.det [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ﻿[Public]
 Object=(Name=XOpenGLDrv.XOpenGLRenderDevice,Class=Class,MetaClass=Engine.RenderDevice,Autodetect=opengl32.dll)
-Preferences=(Caption="XOpenGL Support",Parent="Rendering",Class=XOpenGLDrv.XOpenGLRenderDevice,Immediate=True, Category="Options")
-Preferences=(Caption="Debug Options",Parent="XOpenGL Support",Class=XOpenGLDrv.XOpenGLRenderDevice,Immediate=True, Category="DebugOptions")
-Preferences=(Caption="Render Options",Parent="XOpenGL Support",Class=XOpenGLDrv.XOpenGLRenderDevice,Immediate=True, Category="Client")
+Preferences=(Caption="XOpenGL Support",Parent="Rendering",LangCaption="XOpenGL-Video",Class=XOpenGLDrv.XOpenGLRenderDevice,Immediate=True, Category="Options")
+Preferences=(Caption="Debug Options",Parent="XOpenGL Support",LangCaption="Optionen zum Debuggen",Class=XOpenGLDrv.XOpenGLRenderDevice,Immediate=True, Category="DebugOptions")
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/XOpenGLDrv.det:4]:
```suggestion
Preferences=(Caption="Debug Options",Parent="XOpenGL Support",LangCaption="Debug-Optionen",Class=XOpenGLDrv.XOpenGLRenderDevice,Immediate=True, Category="DebugOptions")
```

File: Unreal/SystemLocalized/det/XOpenGLDrv.det [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
+Preferences=(Caption="Render Options",Parent="XOpenGL Support",LangCaption="Optionen zum Rendern",Class=XOpenGLDrv.XOpenGLRenderDevice,Immediate=True, Category="Client")
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/XOpenGLDrv.det:5]:
```suggestion
Preferences=(Caption="Render Options",Parent="XOpenGL Support",LangCaption="Render-Optionen",Class=XOpenGLDrv.XOpenGLRenderDevice,Immediate=True, Category="Client")
```

**Comment by ividyon** [comment]
> "Mit Hauptserver verbinden" -> "Hauptserver Verbindung" or "Master Server Verbindung"

@JanUrbansky Is this supposed to be the other way around? These are the commonly known "Deppenleerzeichen" and I'd avoid them at all cost. I couldn't see either string in the commits though, so I'm not sure what the comment is referring to.

File: Unreal/SystemLocalized/det/UnrealIntegrity.det (hunk 2 above)
Comment (ividyon) [inline Unreal/SystemLocalized/det/UnrealIntegrity.det:4]:
```suggestion
Preferences=(Caption="Whitelist",Parent="Unreal Integrity",LangCaption="Whitelist",Class=UnrealIntegrity.ExternalWhiteList,Immediate=True)
```

File: Unreal/SystemLocalized/det/UnrealIntegrity.det [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ﻿[Public]
-Preferences=(Caption="Unreal Integrity",Parent="Networking")
-Preferences=(Caption="Integrity Server",Parent="Unreal Integrity",Class=UnrealIntegrity.IntegrityServer,Immediate=True)
-Preferences=(Caption="Whitelist",Parent="Unreal Integrity",Class=UnrealIntegrity.ExternalWhiteList,Immediate=True)
+Preferences=(Caption="Unreal Integrity",Parent="Networking",LangCaption="Integrität von Unreal")
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/UnrealIntegrity.det:2]:
```suggestion
Preferences=(Caption="Unreal Integrity",Parent="Networking",LangCaption="Unreal Integrity")
```

"Unreal Integrity" is the given name of the feature, it is not some common concept that needs translation.

File: Unreal/SystemLocalized/det/Engine.det [hunk 6]
```diff
@@ -4,42 +4,32 @@ Object=(Name=Engine.ServerCommandlet,Class=Class,MetaClass=Core.Commandlet)
...
+Preferences=(Caption="Joystick",Parent="Advanced Options",LangCaption="Joystick")
+Preferences=(Caption="Drivers",Parent="Advanced Options",LangCaption="Treiber",Class=Engine.Engine,Immediate=False,Category=Drivers)
+Preferences=(Caption="Game Settings",Parent="Advanced Options",LangCaption="Spieleinstellungen",Class=Engine.GameInfo,Immediate=True)
+Preferences=(Caption="Decals",Parent="Advanced Options",LangCaption="Marken")
 ; Advanced
-Preferences=(Caption="Game Engine Settings",Parent="Advanced",Class=Engine.GameEngine,Category=Settings,Immediate=True)
-Preferences=(Caption="Key Aliases",Parent="Advanced",Class=Engine.Input,Immediate=True,Category=Aliases)
-Preferences=(Caption="Raw Key Bindings",Parent="Advanced",Class=Engine.Input,Immediate=True,Category=RawKeys)
+Preferences=(Caption="Game Engine Settings",Parent="Advanced",LangCaption="Game-Engine-Einstellungen",Class=Engine.GameEngine,Category=Settings,Immediate=True)
+Preferences=(Caption="Key Aliases",Parent="Advanced",LangCaption="Schaltflächenbefehle",Class=Engine.Input,Immediate=True,Category=Aliases)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/Engine.det:19]:
```suggestion
Preferences=(Caption="Key Aliases",Parent="Advanced",LangCaption="Tastenbefehle",Class=Engine.Input,Immediate=True,Category=Aliases)
```

File: Unreal/SystemLocalized/det/Engine.det [hunk 7]
```diff
@@ -4,42 +4,32 @@ Object=(Name=Engine.ServerCommandlet,Class=Class,MetaClass=Core.Commandlet)
...
+Preferences=(Caption="Raw Key Bindings",Parent="Advanced",LangCaption="Tastenfunktionen",Class=Engine.Input,Immediate=True,Category=RawKeys)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/Engine.det:20]:
```suggestion
Preferences=(Caption="Raw Key Bindings",Parent="Advanced",LangCaption="Rohe Tastenbelege",Class=Engine.Input,Immediate=True,Category=RawKeys)
```

File: Unreal/SystemLocalized/det/Engine.det [hunk 8]
```diff
@@ -4,42 +4,32 @@ Object=(Name=Engine.ServerCommandlet,Class=Class,MetaClass=Core.Commandlet)
...
 ; Networking
-Preferences=(Caption="Public Server Information",Parent="Networking",Class=Engine.GameReplicationInfo,Immediate=True)
-Preferences=(Caption="Channel Download",Parent="Networking",Class=Engine.ChannelDownload)
-Preferences=(Caption="Connection Notifications",Parent="Networking",Class=Engine.Gameinfo,Category=Networking,Immediate=True)
-Preferences=(Caption="Map Lists",Parent="Networking")
+Preferences=(Caption="Public Server Information",Parent="Networking",LangCaption="Öffentliche Server-Info",Class=Engine.GameReplicationInfo,Immediate=True)
+Preferences=(Caption="Channel Download",Parent="Networking",LangCaption="Kanal Herunterladen",Class=Engine.ChannelDownload)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/Engine.det:23]:
```suggestion
Preferences=(Caption="Channel Download",Parent="Networking",LangCaption="Kanal-Download",Class=Engine.ChannelDownload)
```

File: Unreal/SystemLocalized/det/Engine.det [hunk 9]
```diff
@@ -4,42 +4,32 @@ Object=(Name=Engine.ServerCommandlet,Class=Class,MetaClass=Core.Commandlet)
...
+Preferences=(Caption="Connection Notifications",Parent="Networking",LangCaption="Verbindungsbenachrichtigungen",Class=Engine.Gameinfo,Category=Networking,Immediate=True)
+Preferences=(Caption="Map Lists",Parent="Networking",LangCaption="Kartenlisten")
 ; Decals
-Preferences=(Caption="Blood",Parent="Decals")
-Preferences=(Caption="Pawn Shadow",Parent="Decals",Class=Engine.PawnShadow,Immediate=True)
-Preferences=(Caption="Decoration Shadow",Parent="Decals",Class=Engine.DecoShadow,Immediate=True)
-Preferences=(Caption="Projectile Shadow",Parent="Decals",Class=Engine.ProjectileShadow,Immediate=True)
+Preferences=(Caption="Blood",Parent="Decals",LangCaption="Blut")
+Preferences=(Caption="Pawn Shadow",Parent="Decals",LangCaption="Spielerschatten",Class=Engine.PawnShadow,Immediate=True)
+Preferences=(Caption="Decoration Shadow",Parent="Decals",LangCaption="Deko Schatten",Class=Engine.DecoShadow,Immediate=True)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/Engine.det:29]:
```suggestion
Preferences=(Caption="Decoration Shadow",Parent="Decals",LangCaption="Dekorationsschatten",Class=Engine.DecoShadow,Immediate=True)
```

File: Unreal/SystemLocalized/det/Engine.det [hunk 10]
```diff
@@ -4,42 +4,32 @@ Object=(Name=Engine.ServerCommandlet,Class=Class,MetaClass=Core.Commandlet)
...
+Preferences=(Caption="Channel Download",Parent="Networking",LangCaption="Kanal Herunterladen",Class=Engine.ChannelDownload)
+Preferences=(Caption="Connection Notifications",Parent="Networking",LangCaption="Verbindungsbenachrichtigungen",Class=Engine.Gameinfo,Category=Networking,Immediate=True)
+Preferences=(Caption="Map Lists",Parent="Networking",LangCaption="Kartenlisten")
 ; Decals
-Preferences=(Caption="Blood",Parent="Decals")
-Preferences=(Caption="Pawn Shadow",Parent="Decals",Class=Engine.PawnShadow,Immediate=True)
-Preferences=(Caption="Decoration Shadow",Parent="Decals",Class=Engine.DecoShadow,Immediate=True)
-Preferences=(Caption="Projectile Shadow",Parent="Decals",Class=Engine.ProjectileShadow,Immediate=True)
+Preferences=(Caption="Blood",Parent="Decals",LangCaption="Blut")
+Preferences=(Caption="Pawn Shadow",Parent="Decals",LangCaption="Spielerschatten",Class=Engine.PawnShadow,Immediate=True)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/Engine.det:28]:
```suggestion
Preferences=(Caption="Pawn Shadow",Parent="Decals",LangCaption="Figurschatten",Class=Engine.PawnShadow,Immediate=True)
```

File: Unreal/SystemLocalized/det/Engine.det [hunk 11]
```diff
@@ -4,42 +4,32 @@ Object=(Name=Engine.ServerCommandlet,Class=Class,MetaClass=Core.Commandlet)
...
+Preferences=(Caption="Decoration Shadow",Parent="Decals",LangCaption="Deko Schatten",Class=Engine.DecoShadow,Immediate=True)
+Preferences=(Caption="Projectile Shadow",Parent="Decals",LangCaption="Projektil Schatten",Class=Engine.ProjectileShadow,Immediate=True)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/Engine.det:30]:
```suggestion
Preferences=(Caption="Projectile Shadow",Parent="Decals",LangCaption="Projektilschatten",Class=Engine.ProjectileShadow,Immediate=True)
```

File: Unreal/SystemLocalized/det/IpDrv.det [hunk 12]
```diff
@@ -5,8 +5,8 @@ Object=(Name=IpDrv.CompressCommandlet,Class=Class,MetaClass=Core.Commandlet)
 Object=(Name=IpDrv.DecompressCommandlet,Class=Class,MetaClass=Core.Commandlet)
 Object=(Name=IpDrv.TcpNetDriver,Class=Class,MetaClass=Engine.NetDriver)
 Object=(Name=IpDrv.UdpBeacon,Class=Class,MetaClass=Engine.Actor)
-Preferences=(Caption="TCP/IP Network Play",Parent="Networking",Class=IpDrv.TcpNetDriver)
-Preferences=(Caption="Server Beacon",Parent="Networking",Class=IpDrv.UdpBeacon,Immediate=True)
+Preferences=(Caption="TCP/IP Network Play",Parent="Networking",LangCaption="TCP/IP-Netzwerkspiel",Class=IpDrv.TcpNetDriver)
+Preferences=(Caption="Server Beacon",Parent="Networking",LangCaption="Server-Broadcast",Class=IpDrv.UdpBeacon,Immediate=True)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/IpDrv.det:9]:
```suggestion
Preferences=(Caption="Server Beacon",Parent="Networking",LangCaption="Server-Beacon",Class=IpDrv.UdpBeacon,Immediate=True)
```
Translating an English term with a completely different English term would be very misleading and these seem to have nothing to do with one another.

File: Unreal/SystemLocalized/det/IpServer.det [hunk 13]
```diff
@@ -1,6 +1,6 @@
 ﻿[Public]
 Object=(Name=IpServer.UdpServerUplink,Class=Class,MetaClass=Engine.Actor)
-Preferences=(Caption="Master Server Uplink",Parent="Networking",Class=IpServer.UdpServerUplink,Immediate=True)
+Preferences=(Caption="Master Server Uplink",Parent="Networking",LangCaption="Mit Hauptserver verbinden",Class=IpServer.UdpServerUplink,Immediate=True)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/IpServer.det:3]:
```suggestion
Preferences=(Caption="Master Server Uplink",Parent="Networking",LangCaption="Masterserver-Uplink",Class=IpServer.UdpServerUplink,Immediate=True)
```

File: Unreal/SystemLocalized/det/SoftDrv.det [hunk 14]
```diff
@@ -1,6 +1,6 @@
 ﻿[Public]
 Object=(Name=SoftDrv.SoftwareRenderDevice,Class=Class,MetaClass=Engine.RenderDevice)
-Preferences=(Caption="Software Rendering",Parent="Rendering",Class=SoftDrv.SoftwareRenderDevice,Immediate=True)
+Preferences=(Caption="Software Rendering",Parent="Rendering",LangCaption="Software-Rendern",Class=SoftDrv.SoftwareRenderDevice,Immediate=True)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/SoftDrv.det:3]:
```suggestion
Preferences=(Caption="Software Rendering",Parent="Rendering",LangCaption="Software-Video",Class=SoftDrv.SoftwareRenderDevice,Immediate=True)
```

File: Unreal/SystemLocalized/det/UnrealShare.det [hunk 15]
```diff
@@ -178,12 +178,12 @@ Object=(Name="ConsoleKey",Class=Class,MetaClass=Engine.Input,Description="Konsol
...
-Preferences=(Caption="Team Play",Parent="Game Types",Class=UnrealShare.TeamGame,Immediate=True)
-Preferences=(Caption="Config",Parent="Decals",Class=UnrealShare.Scorch,Immediate=True,Category=Decal)
-Preferences=(Caption="Client",Parent="Blood",Class=UnrealShare.UnrealBlood,Immediate=True,Category=BloodClient)
-Preferences=(Caption="Deathmatch Maps",Parent="Map Lists",Class=UnrealShare.DmMapList,Immediate=True)
+Preferences=(Caption="Coop Play",Parent="Game Types",LangCaption="Co-op-Spiel",Class=UnrealShare.CoopGame,Immediate=True)
+Preferences=(Caption="Deathmatch",Parent="Game Types",LangCaption="Deathmatch",Class=UnrealShare.DeathmatchGame,Immediate=True)
+Preferences=(Caption="Team Play",Parent="Game Types",LangCaption="Teamspiel",Class=UnrealShare.TeamGame,Immediate=True)
+Preferences=(Caption="Config",Parent="Decals",LangCaption="Konfig",Class=UnrealShare.Scorch,Immediate=True,Category=Decal)
+Preferences=(Caption="Client",Parent="Blood",LangCaption="Klient",Class=UnrealShare.UnrealBlood,Immediate=True,Category=BloodClient)
+Preferences=(Caption="Deathmatch Maps",Parent="Map Lists",LangCaption="Kartenlisten fur Deathmatch",Class=UnrealShare.DmMapList,Immediate=True)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/UnrealShare.det:186]:
```suggestion
Preferences=(Caption="Deathmatch Maps",Parent="Map Lists",LangCaption="Deathmatch-Karten",Class=UnrealShare.DmMapList,Immediate=True)
```

File: Unreal/SystemLocalized/det/UnrealShare.det [hunk 16]
```diff
@@ -178,12 +178,12 @@ Object=(Name="ConsoleKey",Class=Class,MetaClass=Engine.Input,Description="Konsol
...
-Preferences=(Caption="Deathmatch",Parent="Game Types",Class=UnrealShare.DeathmatchGame,Immediate=True)
-Preferences=(Caption="Team Play",Parent="Game Types",Class=UnrealShare.TeamGame,Immediate=True)
-Preferences=(Caption="Config",Parent="Decals",Class=UnrealShare.Scorch,Immediate=True,Category=Decal)
-Preferences=(Caption="Client",Parent="Blood",Class=UnrealShare.UnrealBlood,Immediate=True,Category=BloodClient)
-Preferences=(Caption="Deathmatch Maps",Parent="Map Lists",Class=UnrealShare.DmMapList,Immediate=True)
+Preferences=(Caption="Coop Play",Parent="Game Types",LangCaption="Co-op-Spiel",Class=UnrealShare.CoopGame,Immediate=True)
+Preferences=(Caption="Deathmatch",Parent="Game Types",LangCaption="Deathmatch",Class=UnrealShare.DeathmatchGame,Immediate=True)
+Preferences=(Caption="Team Play",Parent="Game Types",LangCaption="Teamspiel",Class=UnrealShare.TeamGame,Immediate=True)
+Preferences=(Caption="Config",Parent="Decals",LangCaption="Konfig",Class=UnrealShare.Scorch,Immediate=True,Category=Decal)
+Preferences=(Caption="Client",Parent="Blood",LangCaption="Klient",Class=UnrealShare.UnrealBlood,Immediate=True,Category=BloodClient)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/UnrealShare.det:185]:
```suggestion
Preferences=(Caption="Client",Parent="Blood",LangCaption="Client",Class=UnrealShare.UnrealBlood,Immediate=True,Category=BloodClient)
```

File: Unreal/SystemLocalized/det/UnrealShare.det [hunk 17]
```diff
@@ -178,12 +178,12 @@ Object=(Name="ConsoleKey",Class=Class,MetaClass=Engine.Input,Description="Konsol
...
-Preferences=(Caption="Coop Play",Parent="Game Types",Class=UnrealShare.CoopGame,Immediate=True)
-Preferences=(Caption="Deathmatch",Parent="Game Types",Class=UnrealShare.DeathmatchGame,Immediate=True)
-Preferences=(Caption="Team Play",Parent="Game Types",Class=UnrealShare.TeamGame,Immediate=True)
-Preferences=(Caption="Config",Parent="Decals",Class=UnrealShare.Scorch,Immediate=True,Category=Decal)
-Preferences=(Caption="Client",Parent="Blood",Class=UnrealShare.UnrealBlood,Immediate=True,Category=BloodClient)
-Preferences=(Caption="Deathmatch Maps",Parent="Map Lists",Class=UnrealShare.DmMapList,Immediate=True)
+Preferences=(Caption="Coop Play",Parent="Game Types",LangCaption="Co-op-Spiel",Class=UnrealShare.CoopGame,Immediate=True)
+Preferences=(Caption="Deathmatch",Parent="Game Types",LangCaption="Deathmatch",Class=UnrealShare.DeathmatchGame,Immediate=True)
+Preferences=(Caption="Team Play",Parent="Game Types",LangCaption="Teamspiel",Class=UnrealShare.TeamGame,Immediate=True)
+Preferences=(Caption="Config",Parent="Decals",LangCaption="Konfig",Class=UnrealShare.Scorch,Immediate=True,Category=Decal)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/UnrealShare.det:184]:
```suggestion
Preferences=(Caption="Config",Parent="Decals",LangCaption="Einstellung",Class=UnrealShare.Scorch,Immediate=True,Category=Decal)
```

**Comment by NeonKnightOA** [comment]
I think I've tackled everything suggested. How does it look now? Is it ready to go?

File: Unreal/SystemLocalized/det/Engine.det [hunk 18]
```diff
@@ -4,42 +4,32 @@ Object=(Name=Engine.ServerCommandlet,Class=Class,MetaClass=Core.Commandlet)
...
+Preferences=(Caption="Game Engine Settings",Parent="Advanced",LangCaption="Game-Engine-Einstellungen",Class=Engine.GameEngine,Category=Settings,Immediate=True)
+Preferences=(Caption="Key Aliases",Parent="Advanced",LangCaption="Tastenkürzel",Class=Engine.Input,Immediate=True,Category=Aliases)
+Preferences=(Caption="Raw Key Bindings",Parent="Advanced",LangCaption="Tastenbelegung",Class=Engine.Input,Immediate=True,Category=RawKeys)
 ; Networking
-Preferences=(Caption="Public Server Information",Parent="Networking",Class=Engine.GameReplicationInfo,Immediate=True)
-Preferences=(Caption="Channel Download",Parent="Networking",Class=Engine.ChannelDownload)
-Preferences=(Caption="Connection Notifications",Parent="Networking",Class=Engine.Gameinfo,Category=Networking,Immediate=True)
-Preferences=(Caption="Map Lists",Parent="Networking")
+Preferences=(Caption="Public Server Information",Parent="Networking",LangCaption="Öffentliche Server-Info",Class=Engine.GameReplicationInfo,Immediate=True)
+Preferences=(Caption="Channel Download",Parent="Networking",LangCaption="Kanal Download",Class=Engine.ChannelDownload)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/Engine.det:23]:
```suggestion
Preferences=(Caption="Channel Download",Parent="Networking",LangCaption="Kanal-Download",Class=Engine.ChannelDownload)
```

File: Unreal/SystemLocalized/det/Engine.det (hunk 6 above)
Comment (NeonKnightOA) [reply Unreal/SystemLocalized/det/Engine.det]: Applied.

File: Unreal/SystemLocalized/det/Engine.det (hunk 11 above)
Comment (NeonKnightOA) [reply Unreal/SystemLocalized/det/Engine.det]: Applied.

File: Unreal/SystemLocalized/det/IpServer.det (hunk 13 above)
Comment (NeonKnightOA) [reply Unreal/SystemLocalized/det/IpServer.det]: Which one should I keep? "Masterserver-Verbindung" or "Masterserver-Uplink"

File: Unreal/SystemLocalized/det/UnrealI.det (hunk 1 above)
Comment (NeonKnightOA) [reply Unreal/SystemLocalized/det/UnrealI.det]: Reverted the change to the original string.

File: Unreal/SystemLocalized/det/IpServer.det [hunk 19]
```diff
@@ -1,10 +1,10 @@
 ﻿[Public]
 Object=(Name=IpServer.UdpServerUplink,Class=Class,MetaClass=Engine.Actor)
-Preferences=(Caption="Master Server Uplink",Parent="Networking",Class=IpServer.UdpServerUplink,Immediate=True)
+Preferences=(Caption="Master Server Uplink",Parent="Networking",LangCaption="Master Server Verbindung",Class=IpServer.UdpServerUplink,Immediate=True)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/IpServer.det:3]:
```suggestion
Preferences=(Caption="Master Server Uplink",Parent="Networking",LangCaption="Masterserver-Uplink",Class=IpServer.UdpServerUplink,Immediate=True)
```

File: Unreal/SystemLocalized/det/IpServer.det [hunk 20]
```diff
@@ -1,10 +1,10 @@
...
 
 [ServerUplink]
 ; EN: ClassCaption="Master Server Uplink"
-ClassCaption="Mit Hauptserver verbinden"
+ClassCaption="Master Server Verbindung"
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/IpServer.det:7]:
```suggestion
ClassCaption="Masterserver-Uplink"
```

File: Unreal/SystemLocalized/det/UnrealIntegrity.det [hunk 21]
```diff
@@ -1,7 +1,7 @@
 ﻿[Public]
-Preferences=(Caption="Unreal Integrity",Parent="Networking")
-Preferences=(Caption="Integrity Server",Parent="Unreal Integrity",Class=UnrealIntegrity.IntegrityServer,Immediate=True)
-Preferences=(Caption="Whitelist",Parent="Unreal Integrity",Class=UnrealIntegrity.ExternalWhiteList,Immediate=True)
+Preferences=(Caption="Unreal Integrity",Parent="Networking",LangCaption="Unreal Integrity")
+Preferences=(Caption="Integrity Server",Parent="Unreal Integrity",LangCaption="Integritätsserver",Class=UnrealIntegrity.IntegrityServer,Immediate=True)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/UnrealIntegrity.det:3]:
```suggestion
Preferences=(Caption="Integrity Server",Parent="Unreal Integrity",LangCaption="Integrity-Server",Class=UnrealIntegrity.IntegrityServer,Immediate=True)
```

File: Unreal/SystemLocalized/det/Engine.det [hunk 22]
```diff
@@ -4,42 +4,32 @@ Object=(Name=Engine.ServerCommandlet,Class=Class,MetaClass=Core.Commandlet)
...
+Preferences=(Caption="Drivers",Parent="Advanced Options",LangCaption="Treiber",Class=Engine.Engine,Immediate=False,Category=Drivers)
+Preferences=(Caption="Game Settings",Parent="Advanced Options",LangCaption="Spieleinstellungen",Class=Engine.GameInfo,Immediate=True)
+Preferences=(Caption="Decals",Parent="Advanced Options",LangCaption="Marken")
 ; Advanced
-Preferences=(Caption="Game Engine Settings",Parent="Advanced",Class=Engine.GameEngine,Category=Settings,Immediate=True)
-Preferences=(Caption="Key Aliases",Parent="Advanced",Class=Engine.Input,Immediate=True,Category=Aliases)
-Preferences=(Caption="Raw Key Bindings",Parent="Advanced",Class=Engine.Input,Immediate=True,Category=RawKeys)
+Preferences=(Caption="Game Engine Settings",Parent="Advanced",LangCaption="Game-Engine-Einstellungen",Class=Engine.GameEngine,Category=Settings,Immediate=True)
+Preferences=(Caption="Key Aliases",Parent="Advanced",LangCaption="Tastenbefehle",Class=Engine.Input,Immediate=True,Category=Aliases)
+Preferences=(Caption="Raw Key Bindings",Parent="Advanced",LangCaption="Rohe Tastenbelege",Class=Engine.Input,Immediate=True,Category=RawKeys)
```
Comment (ividyon) [inline Unreal/SystemLocalized/det/Engine.det:20]:
```suggestion
Preferences=(Caption="Raw Key Bindings",Parent="Advanced",LangCaption="Rohe Tastenbelegung",Class=Engine.Input,Immediate=True,Category=RawKeys)
```

**Review by ividyon — CHANGES_REQUESTED** [review]

File: Unreal/SystemLocalized/det/IpServer.det (hunk 13 above)
Comment (NeonKnightOA) [reply Unreal/SystemLocalized/det/IpServer.det]: Done then.

**Comment by NeonKnightOA** [comment]
I'm committing this as it is. Looks like we have an imminent release. Hopefully.

Thank you all! 🍻

**Pull request merged**